source/content/guides/disaster-recovery/03-site-goes-down.md

```diff
@@ -19,18 +19,18 @@ integration: [--]
 
 ## Open a Support Ticket
 
-In cases of downtime or significant functional failure in the Live environment, the first step is to open a support ticket. Even if you escalate the incident to your dedicated Customer Success Manager (included for all Enterprise contract customers), our support engineers will be the ones diagnosing the cause of downtime and working to get your site back up, and a ticket is the fastest way to get them up to speed and engaged.
+In cases of downtime or significant functional failure in the live environment, the first step is to open a support ticket. Even if you escalate the incident to your dedicated Customer Success Manager (available to all Enterprise contract customers), our engineering team will be the ones diagnosing the cause of the downtime and working to restore your site. Opening a ticket is the fastest way to get them up to speed and engaged.
 
-* **Diamond** and **Platinum** Account customers can report and escalate site downtime by clicking **Trigger Pantheon On-Call** from the Support tab.
-  * In cases where the dashboard is inaccessible, a ticket can be filed using a telephone ticketing service, accessible at **1(866)415-7624**. Note that this is strictly for filing a ticket, and you will not reach a support engineer by using this method.
-* **Gold** customers should click **Open Ticket** to open a support interaction to report site downtime.
-* **Silver** customers should click **Launch Chat**.
+* **Diamond** and **Platinum** customers can open an **emergency ticket** to report and escalate business-critical site downtime, from the Support tab in the workspace of the affected site.
+  * In cases where the dashboard is inaccessible, a support ticket can be filed using a telephone ticketing service, accessible at 1(866)415-7624. Note that this is strictly for filing a ticket, and you will not reach our engineering team by using this method.
+* **Gold** customers can open a **general support ticket** to report site downtime.
+* **Silver** customers can start a **live chat** to report site downtime.
 
 ![Show platinum support features in the site dashboard](../../../images/dashboard/new-dashboard/platinum-support-site-dashboard.png)
 
-Please include as much information as possible. A support engineer will work with you to diagnose the cause, and any information that you can provide will shorten the investigation time.
+Please provide as much detail as possible. Our engineering team will work with you to diagnose the cause of the issue, and any information you provide will help shorten the investigation time.
```

Comment:
I don't believe we want to conflate our "support team" with our "engineering team", which are two separate things to customers. Support handles tickets, while engineering works on Pantheon itself.
> Please provide as much detail as possible. Our support team will work with you to diagnose [...]

Reply:
+1 to support team . Agreed that 'engineering team' might confuse users on who they are actually speaking with.

Reply:
In this case, should be keep it as "support engineer"?

Reply:
Sounds good to me, we just need to be very cautious to ever refer to "engineering" without qualifying it as "support engineering" (in regards to tickets). Tickets can never go directly to our "engineering" team.